include/SDL3/SDL_ioasync.h

```diff
@@ -0,0 +1,500 @@
+/*
+  Simple DirectMedia Layer
+  Copyright (C) 1997-2024 Sam Lantinga <slouken@libsdl.org>
+
+  This software is provided 'as-is', without any express or implied
+  warranty.  In no event will the authors be held liable for any damages
+  arising from the use of this software.
+
+  Permission is granted to anyone to use this software for any purpose,
+  including commercial applications, and to alter it and redistribute it
+  freely, subject to the following restrictions:
+
+  1. The origin of this software must not be misrepresented; you must not
+     claim that you wrote the original software. If you use this software
+     in a product, an acknowledgment in the product documentation would be
+     appreciated but is not required.
+  2. Altered source versions must be plainly marked as such, and must not be
+     misrepresented as being the original software.
+  3. This notice may not be removed or altered from any source distribution.
+*/
+
+/* WIKI CATEGORY: IOAsync */
+
+/**
+ * # CategoryIOAsync
+ *
+ * SDL offers a way to perform i/o _asynchronously_. This allows an app to
```

Comment:
```suggestion
 * SDL offers a way to perform I/O asynchronously. This allows an app to
```
Also switch i/o to I/O through the rest of this file.

Reply:
Fixed in 6e5e55d01c1f9e12626d00881fd8d9661abda8d8.